typescript/__tests__/parsers/hf/hf.test.ts

```diff
@@ -257,7 +257,7 @@ describe("HuggingFaceTextGeneration ModelParser", () => {
 
     const expectedOutput = {
       output_type: "execute_result",
-      data: "Test text generation",
+      data: { kind: "string", value: "Test text generation" },
```

Comment:
I feel like even though this is slightly easier for the frontend to parse, it makes the model parsers and readability of the aiconfig more annoying. It should be pretty simple to check `typeof output.data === "string"` to infer string type, no? Up to you @rholinshead

Reply:
Sg, let me do this another diff to keep things simple and easier to review!